lasio/reader.py

```diff
@@ -705,18 +705,25 @@ def __init__(self, title, version=1.2):
             self.func = self.metadata
             self.section_name2 = "Version"
         else:
-            raise KeyError("Unknown section name {}".format(title.upper()))
+            # raise KeyError("Unknown section name {}".format(title.upper()))
+            print("Unknown section name {}".format(title.upper()))
```

Comment:
We should probably use `logger.info` instead of `print` here. ?

Reply:
Yes, you are right about logger.info.  I'll change it and post an update.

Reply:
commit 7196665 has the change to use logger.info.